client/src/components/Modal/ModalContainer.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
```

Comment:
Do you mind putting an underscore in the beginning of the file name here? I forgot to mention that when we spoke on slack. That's a way of signalling to other developers that this is a "hidden" file/class/whatever. That will make it extra clear that people shouldn't use this component

Reply:
Just to the file name